App.tsx

```diff
@@ -1,57 +1,56 @@
-import { StatusBar } from 'expo-status-bar';
+import * as React from 'react';
 import { Platform, StyleSheet, Text, View } from 'react-native';
-
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
+import LoginScreen from './src/screens/login';
+import SignupScreen from "./src/screens/signup";
+import { StatusBar } from 'expo-status-bar';
+
 import ClassroomFinderScreen from './src/screens/classroomFinder';
 import Header from './shared/header';
 import { Ionicons  } from '@expo/vector-icons';
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+
 const Stack = createStackNavigator();
 
 type RootStackParamList = {
   classroomFinderScreen: undefined;
 };
-
 const Tab = createBottomTabNavigator();
 
 function Classroom() {
   return (
     <Tab.Navigator
-        screenOptions={({ route }) => ({
-          headerShown: false,
-          tabBarIcon: ({ focused, color, size }) => {
-            let iconName:any;
+      screenOptions={({ route }) => ({
+        headerShown: false,
+        tabBarIcon: ({ focused, color, size }) => {
+          let iconName:any;
 
-            if (route.name === 'Home') {
-              iconName = 'bookmark'
-            } else if (route.name === 'Settings') {
-              iconName = 'ios-list';
-            }
+          if (route.name === 'Home') {
+            iconName = 'bookmark'
+          } else if (route.name === 'Settings') {
+            iconName = 'ios-list';
+          }
 
-            // You can return any component that you like here!
-            return <Ionicons name={iconName} size={24} color="#037A87" />;
-          },
-          tabBarActiveTintColor: 'tomato',
-          tabBarInactiveTintColor: 'gray',
-        })}
-      >
-        <Tab.Screen name="Home" component={ClassroomFinderScreen} />
-      </Tab.Navigator>
+          // You can return any component that you like here!
+          return <Ionicons name={iconName} size={24} color="#037A87" />;
+        },
+        tabBarActiveTintColor: 'tomato',
+        tabBarInactiveTintColor: 'gray',
+      })}
+    >
+      <Tab.Screen name="Home" component={ClassroomFinderScreen} />
+    </Tab.Navigator>
   )
 }
 
+
 export default function App() {
   return (
-      <NavigationContainer>
-      <Stack.Navigator
-       initialRouteName='classroomFinderScreen'
-       screenOptions={{
-        header: () => <Header />
-      }}>
-        <Stack.Screen 
-          name = "classroomFinderScreen"
-          component={Classroom}/>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName={"Login"} screenOptions={{header: () => <Header />}}>
+        <Stack.Screen name={"Login"} component={LoginScreen} options={{title: 'Login', headerShown:false}} />
+        <Stack.Screen name={"Signup"} component={SignupScreen} options={{title: 'Signup', headerShown: false}} />
```

Comment:
To navigate to classroom finder screen we need to keep the Stack.Screen that has Classroom component